spec/lib/api/2.0/lookups-spec.js

```diff
@@ -0,0 +1,130 @@
+// Copyright 2016, EMC, Inc.
+/* jshint node:true */
+
+'use strict';
+
+describe('Http.Api.Lookup 2.0', function () {
+    var waterline, stub;
+
+    var data = [
+        {
+            id: '123',
+            createdAt: 'createdAt',
```

Comment:
Even this is unit-test and this is correct to meet the test logic, but I still suggest using a real datetime string. Because the unit-test is a kind of code document, developer may lookup the unit-test to find how to use the API.